src/v2/Apps/Consign/Components/FAQ.tsx

```diff
@@ -1,29 +1,205 @@
-import React from "react"
-import { Box, Text, Toggle } from "@artsy/palette"
+import React, { useState } from "react"
+import { Box, Clickable, Flex, Text } from "@artsy/palette"
+import { SectionContainer } from "./SectionContainer"
 
 export const FAQ: React.FC = () => {
   return (
-    <Box>
+    <SectionContainer>
+      <Text width="100%" textAlign="left" mb={4} variant="largeTitle">
+        Frequently Asked Questions Read more about selling your artwork
+      </Text>
       <Box>
-        <Text variant="title">Frequently Asked Questions</Text>
-      </Box>
-      <Box>
-        <Toggle label="How does selling on Artsy work?">
-          <Text variant="text">
+        <ToggleButton label="How does selling on Artsy work?">
+          <TextItem>
             Artsy helps you find the best sales solution for works in your
             collection. We review the works you’re interested in selling,
             determine the strength of the secondary market demand, and suggest
-            an appropriate sales strategy. This can involve matching you with
-            the right seller (e.g., an auction house or a gallery), selling
-            qualified works via Artsy’s online marketplace, or selling the work
-            privately through our network of trusted partners and collectors.
+            an appropriate sales strategy. <br />
```

Comment:
why we don't have proper `<p>` tags in force is a bit of a mystery, but messing with that right now is just not possible without an army of QA